gensim/test/test_ldamallet_wrapper.py

```diff
@@ -10,7 +10,13 @@
 
 
 import logging
-import unittest2 as unittest
+import sys
+
+if sys.version_info < (2, 7):
```

Comment:
@souravsingh now we don't support Python version that older than 2.7, you no need this `if`